recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.0.3" %}
+{% set version = "4.2.11" %}
```

Comment:
This won't work b/c a feedstock already exists. Please send a PR to the feedstock maintainance branch: https://github.com/conda-forge/django-feedstock/tree/4.x

Reply:
Thank you, I dont know how I missed that branch.
 - https://github.com/conda-forge/django-feedstock/pull/176

Reply:
I just created it :-)

But we don't add old versions as feedstock, always as branches in existing ones.